lms/djangoapps/ccx/utils.py

```diff
@@ -268,20 +273,17 @@ def ccx_students_enrolling_center(action, identifiers, email_students, course_ke
                 errors.append(u"{0}".format(exp))
                 continue
 
-            # Check the License limit in the pearson course operation plugin.
-            if (
-                    configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False)
-                    and is_ccx_course(course_key)
-                    and not must_enroll
-               ):
-                allow_enrollment, message = run_extension_point(
+            if is_course_licensing_enable and not must_enroll:
+                # Check the License limit in the pearson course operation plugin.
+                allow_enrollment = run_extension_point(
```

Comment:
`allow_enrollment` Variable used once.

```suggestion
                if is_course_licensing_enable and not must_enroll:
                  # Check the License limit in the pearson course operation plugin.
                  if not run_extension_point(
                      'PCO_ENFORCE_LICENSE_LIMITS',
                      course_key=course_key,
                      email=email,
                      student=student,
                      request=current_request,
                  ):
                      break
```